portable/MemMang/heap_5.c

```diff
@@ -343,6 +354,22 @@ size_t xPortGetMinimumEverFreeHeapSize( void )
 }
 /*-----------------------------------------------------------*/
 
+void * pvPortCalloc( size_t xNum,
+                     size_t xSize )
+{
+    void * pv = NULL;
+
+    pv = pvPortMalloc( xNum * xSize );
+
+    if( pv != NULL )
+    {
+        memset( pv, 0, xNum * xSize );

```

Comment:
memset return value is not used here. Should it be cast to null?

Reply:
Good point. Will do.